### ISSUE TITLE
Rule Change: Make relative-font-units unit matching case-insensitive

### ISSUE THREAD

**Issue by xbinaryx** [eslint/css#215]
### What rule do you want to change?

relative-font-units

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new default behavior

### Example code

```css
/* eslint css/relative-font-units: ["error"] */

.a {
    font-size: 1REM;
}
```

### What does the rule currently do for this code?

It reports a false positive error, saying that only allowed relative units should be used, even though REM is a valid relative unit (CSS units are case-insensitive).

### What will the rule do after it's changed?

It will not report an error for 1REM (or any other case variation of allowed units), correctly recognizing that CSS units are case-insensitive and REM is equivalent to rem.

### Participation

- [x] I am willing to submit a pull request to implement this change.
